tools/ci_build/build.py

```diff
@@ -652,6 +655,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
         "-Donnxruntime_BUILD_WINML_TESTS=" + ("OFF" if args.skip_winml_tests else "ON"),
         "-Donnxruntime_GENERATE_TEST_REPORTS=ON",
         "-Donnxruntime_DEV_MODE=" + use_dev_mode(args),
+        "-DPython_EXECUTABLE=" + sys.executable,
```

Comment:
Why do we need both PYTHON_EXECUTABLE and Python_EXECUTABLE?

Reply:
To be compatible with old code, mainly onnx cmake files.  Once I make a similar change to ONNX, the duplicated line can be removed.

Reply:
Thanks. Can you please make a comment in the file stating this?